tools/munin2snmp/munin2snmp.pl

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/perl
+
+use strict;
+use NetSNMP::OID;
+use NetSNMP::ASN (':all');
+use NetSNMP::agent (':all');
+use IO::Socket;
+
+my %cache = ();         # Cache
+my @cache_oids = ();    # Keys, sorted
+my $cache_updated = 0;
+my $oidbase = ".1.3.6.1.4.1.123456.100.1.1";
+my $delimiter = ' = ';
+
+# initialize
+my %Munin;
+$Munin{PORT} = '4949';
+$Munin{HOST} = 'localhost';
```

Comment:
munin2snmp is a separate program. it doesn't have access to munin env variables as it is not started from munin-node. I've added some code to read a config file or command arguments